docs/index.rst

```diff
@@ -1,19 +1,67 @@
-===============================
-Welcome to the CoronaHackathon!
-===============================
+================================
+Welcome to the Grenz-er-fahrung!
+================================
 
+As part of the WirvsVirus_ hackathon we join the fight against COVID-19.
+
+.. _WirvsVirus: https://wirvsvirushackathon.org/
+
+Introduction
+============
+
+The last couple of weeks were defined by the spread of the virus around the world. Europe, in particular, rapidly became the epicentre of the current epidemic. As a consequence borders throughout Europe were closed in a rush leading to haevy traffic jams. Europe's economy, famous for its free and borderless markets, risks being halted due to merchants, commuters, and truckers being stuck attempting to cross a border.
+
+For this weekend we chose to **hack** together a prototype app which allows officials to provide questionnairies digitally, enabling border crossers to answer questions in advance and reduce the work of border policemen.
 
 Introduction
 ============
-Here we give a short introduction
+
+Consider the following user story:
+
+.. topic:: A truck driver delivering goods throughout Europe
+
+   On his journey the truck driver travels from country A to B but does not speak the language spoken in country B fluently.
+
+   How can the time of the border crossing be minimized?
```

Comment:
Should we perhaps mention this before mentioning the language barriers? Because even without any language barriers it would be possible to reduce time. The language compatibility is a nice add-on I would say. What do you think?

Reply:
Should we perhaps emphasize here again that this issue occurs due to the newly created, fastly established controls, and that technology can be used to help minimize the time?